web/src/main/java/org/openmrs/web/Listener.java

```diff
@@ -423,8 +430,29 @@ private void copyCustomizationIntoWebapp(ServletContext servletContext, Properti
 						}
 					}
 				}
+				PathMatchingResourcePatternResolver patternResolver = new PathMatchingResourcePatternResolver();
```

Comment:
Do you think ServletContextResourcePatternResolver would have reduced the lines of code here?